### ISSUE TITLE
[Bug Report] Popover doesn't close if route changes

### ISSUE THREAD

**Issue by matthagemann** [ElemeFE/element#12187]
### Element UI version
2.4.5

### OS/Browsers version
Latest Chrome

### Vue version
2.5.16

### Reproduction Link
[https://jsfiddle.net/mmx38qxw/4433/](https://jsfiddle.net/mmx38qxw/4433/)

### Steps to reproduce
Change route, the popover won't close.

### What is Expected?
Default user expectation is that popover will close on route change.

### What is actually happening?
Popover not closing.

**Comment by ziyoung** [comment]
I think it isn't a bug of ElementUI. The `router-view` component renders the matched component for the given path. When route changes, only the `router-view` component renders. So the popover still opens.

```
<router-view></router-view>
<el-popover></el-popover>
```

**Comment by matthagemann** [comment]
Makes sense, thanks. How can I trigger a close on click? I have no option but to keep the popover outside of `router-view`.

**Comment by jikkai** [comment]
@matthagemann 
You can control it manually by setting `trigger="manual"`

**Comment by matthagemann** [comment]
@jikkai I cannot find documentation on that, except on popper.js itself: `instance.hide()` and `instance.show()`. How does that work in an Element environment?

**Comment by jikkai** [comment]
@matthagemann Hope this demo can help you :)

https://jsfiddle.net/mmx38qxw/4538/
